lib/active_event_store/subscriber_job.rb

```diff
@@ -41,7 +41,11 @@ def perform(payload)
       event = event_store.deserialize(**payload, serializer: ActiveEventStore.config.serializer)
 
       event_store.with_metadata(**event.metadata.to_h) do
-        subscriber.call(event)
+        if subscriber.is_a?(Class) && !subscriber.respond_to?(:call)
+          subscriber.new.call(event)
+        else
+          subscriber.call(event)
+        end
```

Comment:
Let's move this into the `#subscriber` private method, so the logic of building a subscriber would stay there.